utils/import_srg_spreadsheet.py

```diff
@@ -0,0 +1,256 @@
+
+from ssg.rule_yaml import find_section_lines, get_yaml_contents
+
+SSG_ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+RULES_JSON = os.path.join(SSG_ROOT, "build", "rule_dirs.json")
+SEVERITY = {'CAT III': 'low', 'CAT II': 'medium', 'CAT I': 'high'}
+
+# The start row is 2, to avoid importing the header
+START_ROW = 2
+
+
+class Row:
+    row_id = 0
+    IA_Control = ""
+    CCI = ""
+    SRGID = ""
+    STIGID = ""
+    SRG_Requirement = ""
+    Requirement = ""
+    SRG_VulDiscussion = ""
+    Vul_Discussion = ""
+    Status = ""
+    SRG_Check = ""
+    Check = ""
+    SRG_Fix = ""
+    Fix = ""
+    Severity = ""
+    Mitigation = ""
+    Artifact_Description = ""
+    Status_Justification = ""
+
+    @staticmethod
+    def from_row(sheet: Worksheet, row_number: int, full_name: str, changed_name: str) -> Row:
+        self = Row()
+        self.row_id = row_number
+        self.IA_Control = sheet[f'A{row_number}'].value
+        self.CCI = sheet[f'B{row_number}'].value
+        self.SRGID = sheet[f'C{row_number}'].value
+        self.STIGID = sheet[f'D{row_number}'].value
+        self.SRG_Requirement = sheet[f'E{row_number}'].value
+        self.Requirement = sheet[f'F{row_number}'].value.replace(full_name, changed_name)
+        self.SRG_VulDiscussion = sheet[f'H{row_number}'].value
+        self.Vul_Discussion = sheet[f'G{row_number}'].value
+        self.Status = sheet[f'I{row_number}'].value
+        self.SRG_Check = sheet[f'J{row_number}'].value
+        self.Check = sheet[f'K{row_number}'].value
+        self.SRG_Fix = sheet[f'L{row_number}'].value
+        self.Fix = sheet[f'M{row_number}'].value
+        self.Severity = sheet[f'N{row_number}'].value
+        self.Mitigation = sheet[f'O{row_number}'].value
+        self.Artifact_Description = sheet[f'P{row_number}'].value
+        self.Artifact_Description = sheet[f'Q{row_number}'].value
+        return self
+
+    def __str__(self) -> str:
+        return f'<Row {self.row_id}>'
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--changed', '-c', help="The file with changes, in xlsx", required=True)
+    parser.add_argument('--current', '-b', help="The latest file from CaC, in xlsx", required=True)
+    parser.add_argument('--changed-name', '-n', type=str, action="store",
+                        help="The name that DISA uses for the producut. Defaults to RHEL 9",
+                        default="RHEL 9")
+    parser.add_argument("-r", "--root", type=str, action="store", default=SSG_ROOT,
+                        help=f"Path to SSG root directory (defaults to {SSG_ROOT})")
+    parser.add_argument("-j", "--json", type=str, action="store", default=RULES_JSON,
+                        help=f"Path to the rules_dir.json (defaults to {RULES_JSON})")
+    parser.add_argument("-p", "--product", type=str, action="store", required=True,
+                        help="What product tofix_cac_cells")
```

Comment:
The help text is little weird.

```suggestion
                        help="Product ID")
```